account_banking_sepa_credit_transfer/tests/test_sct.py

```diff
@@ -230,7 +228,6 @@ def test_usd_currency_sct(self):
         self.assertEqual(asus_pay_line1.communication, 'Inv9032')
         self.payment_order.draft2open()
         self.assertEqual(self.payment_order.state, 'open')
-        self.assertEqual(self.payment_order.sepa, False)
```

Comment:
You'll probably want to test the equivalent flag on the line, instead of completely removing the assertion.

Reply:
Right. I added some checks in my commit https://github.com/OCA/bank-payment/pull/658/commits/dcd653f0bb578aa573ed6e6b5f7c924e2b3c4cf8